prow/config/config.go

```diff
@@ -666,6 +670,14 @@ type GitHubOptions struct {
 	LinkURL *url.URL `json:"-"`
 }
 
+// ManagedHmacEntityInfo contains information about the repo/org which is onboarded and associated metadata.
+type ManagedHmacEntityInfo struct {
+	TokenCreatedAfter time.Time `json:"tokenCreatedAfter"`
+}
+
+// ManagedHmacEntity contains information about all the repos/orgs which are onboraded with private tokens.
+type ManagedHmacEntity map[string]ManagedHmacEntityInfo
```

Comment:
nit: If you are going to create a type for this please pluralize it.

Reply:
Done.